### PR TITLE
KIALI-1172 Migrate layout selection to cy toolbar

### PR DESCRIPTION
**Pull request by jshaughn**
- removed from graph settings dropdown
- add new buttons to cy toolbar
- remove unnecessary and poorly named GraphType interface

screenshot

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#45](https://paste.fedoraproject.org/paste/cYg8~W79c1~4yeyCDvQjjQ)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1253]](https://paste.fedoraproject.org/paste/2bh~bv0sA73xYJVZUc152A)
  * :heavy_check_mark: Kiali-on-Openshift [#[597]](https://paste.fedoraproject.org/paste/smx66UkcgA7ubbeYm23rtA)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1024]](https://paste.fedoraproject.org/paste/P~qWdOh4IM1Wv3f4xnF22g)
  * :red_circle: run-kiali-e2e-tests [#[70]](https://paste.fedoraproject.org/paste/OHhWdg1oWnrpgNb-G1Oalw)


KIALI_NAME=kialiuipr670-istio-system

File: src/components/CytoscapeGraph/CytoscapeToolbar.tsx [hunk 1]
```diff
@@ -28,16 +35,52 @@ export class CytoscapeToolbar extends React.PureComponent<CytoscapeToolbarProps>
...
+        <Button
+          onClick={() => {
+            this.props.onLayoutChange(ColaGraph.getLayout());
+          }}
+          className={cytoscapeToolbarPadThinStyle}
+          title={ColaGraph.getLayout().name}
+          active={this.props.activeLayout.name === ColaGraph.getLayout().name}
+        >
+          <div className="fa pficon-topology" /> 2
+        </Button>
```
Comment (josejulio) [inline src/components/CytoscapeGraph/CytoscapeToolbar.tsx]: You probably want to wrap these Buttons in a [ButtonGroup](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Style=default&selectedKind=patternfly-react%2FWidgets%2FButton&selectedStory=ButtonGroup&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)

File: src/pages/Graph/GraphPage.tsx [hunk 2]
```diff
@@ -120,6 +125,17 @@ export default class GraphPage extends React.PureComponent<GraphPageProps> {
     this.scheduleNextPollingInterval(0);
   };
 
+  handleLayoutChange = (layout: Layout) => {
+    console.log('JS\n New Layout! ' + layout.name);
```
Comment (josejulio) [inline src/pages/Graph/GraphPage.tsx]: You probably want to remove this!

File: src/pages/Graph/GraphPage.tsx [hunk 3]
```diff
@@ -120,6 +125,17 @@ export default class GraphPage extends React.PureComponent<GraphPageProps> {
...
+    const params = this.getGraphParams();
+    console.log('JS\nParams=' + JSON.stringify(params));
```
Comment (josejulio) [inline src/pages/Graph/GraphPage.tsx]: > You probably want to remove this!

And this!

**Comment by jshaughn** [comment]
@josejulio The button groups are good and I got rid of the console logging.  Here's an updated the screenshot:

image

**Pull request merged**